README.md

```diff
@@ -39,30 +39,30 @@ The price we pay for beautifully colored displays is complexity required to aggr
 The `cmd2` framework provides a great mixture of both worlds. Application designers can easily create complex applications and rely on the cmd2 library to offer effortless user facing help and extensive tab completion.
 When users become comfortable with functionality, cmd2 turns into a feature rich library enabling a smooth transition to full automation. If designed with enough forethought, a well implemented cmd2 application can serve as a boutique workflow tool. `cmd2` pulls off this flexibility based on two pillars of philosophy:
 
-- Tab Completion
-- Automation Transition
+-   Tab Completion
```

Comment:
All whitespaces like this are due to [prettier](https://prettier.io/) auto-formatting of Markdown and YAML files accoridng to the configuration in the `.prettierrc` configuration file in the `cmd2` project root directory.

There is a `Prettier` extension for VSCode and a `Prettier` plugin for PyCharm - both make it easy to auto-format on save.